### ISSUE TITLE
Does not work for German passport

### ISSUE THREAD

**Issue by psimatis** [ZeroPass/dmrtd#18]
Hi and thanks for this project!

Unfortunately, it does not work for German passports. I ran the code, it detects the chip, but then I get an error.
I assume they use PACE and not BAC anymore (even though I think they should be backwards compatible), or am I missing something?

**Comment by smlu** [comment]
Yes you are right, the German passports don't support BAC protocol anymore. Unfortunately this library doesn't support PACE protocol yet, and that's why you probably get an error.

**Comment by smlu** [comment]
Library now supports PACE protocol (commit be55ab5bef). Please test it and see if you can read German passport now.
